<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QPushButton" name="AddCustomer_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add New customer to waiting list</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>341</width>
     <height>105</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Number</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="inputMask">
       <string>0000000000</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fullname</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Delete Customer from list</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>lineEdit_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>440</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="addname">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Number of customer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="AddNumber">
      <property name="inputMask">
       <string>0000000000</string>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="AddName">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fullname of customer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="maxLength">
       <number>30</number>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="addgroup">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Address of customer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="AddAddress">
      <property name="maxLength">
       <number>25</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="addprice">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Date of getting </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="AddDate"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="AddCustomer">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <bold>true</bold>
        <underline>true</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>AddNewCustomer</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>lineEdit_7</zorder>
   <zorder>addprice</zorder>
   <zorder>addname</zorder>
   <zorder>addgroup</zorder>
   <zorder>AddNumber</zorder>
   <zorder>AddName</zorder>
   <zorder>AddDate</zorder>
   <zorder>AddAddress</zorder>
   <zorder>AddCustomer</zorder>
  </widget>
  <widget class="QPushButton" name="ShowTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Get list of Staff</string>
   </property>
  </widget>
  <widget class="QPushButton" name="List">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show Waiting list</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>aaaaaaaaaaaaaaa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchByNumber">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Search By DepartmentNumber</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteCustomer">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete customer from waiting list</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>1031</width>
     <height>660</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="dragMode">
    <enum>QGraphicsView::NoDrag</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Sort">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>140</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Sort by Department Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchByName">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>70</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Search by Fullname</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>381</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>28</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>24</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <row>
    <property name="text">
     <string>Staff 1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 13</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 14</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Staff 15</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Fullname</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Department Number</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Position</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save Waiting list to a file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>40</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>0000000000</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <zorder>graphicsView</zorder>
  <zorder>formLayoutWidget_3</zorder>
  <zorder>formLayoutWidget</zorder>
  <zorder>tableWidget</zorder>
  <zorder>AddCustomer_2</zorder>
  <zorder>DeleteCustomer</zorder>
  <zorder>List</zorder>
  <zorder>Save</zorder>
  <zorder>SearchByName</zorder>
  <zorder>SearchByNumber</zorder>
  <zorder>ShowTable</zorder>
  <zorder>Sort</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_5</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
